client/src/ingredients/browser-compatibility-table/error-boundary.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+/**
+ * The error boundary for BrowserCompatibilityTable.
+ *
+ * When the whole BrowserCompatibilityTable crashes, for whatever reason,
+ * this component will show a friendly message
+ * to replace that crashed component
+ */
+export class BctErrorBoundary extends React.Component {
+  state = {
+    error: null
+  };
+  componentDidCatch(error, _errorInfo) {
+    this.setState({
+      error
+    });
+    // TODO: Report this error to Sentry, https://github.com/mdn/stumptown-renderer/issues/99
+  }
+  render() {
+    if (this.state.error) {
+      return (
+        <>
+          <h2 id="Browser_compatibility">Browser Compatibility</h2>

```

Comment:
Note to @wbamberg if we do decide to move the labelling of this `h2` to stumptown-content, it will clash with this PR.